clang/lib/Driver/ToolChains/Clang.cpp

```diff
@@ -5584,6 +5584,19 @@ void Clang::ConstructJob(Compilation &C, const JobAction &JA,
         // included, enable the integration header based diagnostics.
         CmdArgs.push_back("-fsycl-enable-int-header-diags");
       }
+
+      // Add the -include-footer option to add the integration footer
+      StringRef Footer = D.getIntegrationFooter(Input.getBaseInput());
+      if (types::getPreprocessedType(Input.getType()) != types::TY_INVALID &&
+          !Args.hasArg(options::OPT_fno_sycl_use_footer) && !Footer.empty()) {
+        CmdArgs.push_back("-include-footer");
+        CmdArgs.push_back(Args.MakeArgString(Footer));
+        // When creating dependency information, filter out the generated
+        // integration footer file.
+        CmdArgs.push_back("-dependency-filter");
+        CmdArgs.push_back(Args.MakeArgString(Footer));
+      }
+
```

Comment:
This and the changes to `test/Driver/sycl-int-footer.cpp` should be taken care of in a separate PR that will cover the entirety of the switch to using `-include-footer` instead of `append-file`.  Performing this now will cause the footer to be added twice.

Reply:
OK. I will remove this code and the test from the PR.